.devcontainer/devcontainer.json

```diff
@@ -11,7 +11,9 @@
 	// To run docker in docker
 	"privileged": true,
 	"features": {
-		"ghcr.io/devcontainers/features/python:1": {}
+		"ghcr.io/devcontainers/features/python:1": {
+			"version": "3.11"
```

Comment:
I though the version was already pin in the dockerfile https://github.com/DataDog/integrations-core/blob/master/.devcontainer/Dockerfile#L3C45-L3C60 

```dockefile
FROM mcr.microsoft.com/devcontainers/python:1-3.11-bullseye
```

Does it means that the feature is installing another python interpreter in the image at start time? 
of it means that we also need to set the version here to have the python feature properly configured?

Reply:
it can be good to add a comment in both file to advertise that if the version is updated in one of the file it needs also to be updated in the other file.